dev-games/tiled/tiled-1.11.0.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs multiprocessing xdg
+
+DESCRIPTION="A general purpose tile map editor"
+HOMEPAGE="https://www.mapeditor.org/"
+SRC_URI="https://github.com/mapeditor/tiled/archive/v${PV}/${P}.tar.gz"
+
+LICENSE="BSD BSD-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="examples wayland headers"
+
+RDEPEND="
+	app-arch/zstd
+	dev-qt/qtbase:6[X]
+	dev-qt/qtimageformats:6
+	dev-qt/qtdeclarative:6
+	wayland? ( dev-qt/qtwayland:6 )
```

Comment:
Is there a build time switch for this or is this relying on automagic?

This would be problematic for binpkgs if its automagic.

Reply:
Just figured out `qtbase` itself pulls in `qtwayland` with `wayland` USE flag enabled, so this dependency isn't even needed.

Reply:
I don't think tiled itself actually uses any of the X or wayland specific qt stuff. Like only mention of wayland in the project is them adding wayland platform plugin support in the snap package. Which would mean its all runtime and therefore not the ebuilds problem.